src/components/AlignmentHost/AlignmentHost.ts

```diff
@@ -269,36 +268,36 @@ namespace fabric {
     private _renderAlignment(): void {
       document.body.appendChild(this._alignmentClone);
     }
-    
-    private _savealignmentSize(): void {
-      let _alignmentStyles = window.getComputedStyle(this._alignmentClone);
-      this._alignmentClone.setAttribute("style", "opacity: 0; z-index: -1");
-      this._alignmentClone.classList.add(ALIGNMENT_STATE_POSITIONED);
-      this._alignmentClone.classList.add(CONTEXT_STATE_CLASS);
-      
-      if(this._matchTargetWidth) {
-        let teStyles = window.getComputedStyle(this._targetElement);
-        this._alignmentWidth = this._targetElement.getBoundingClientRect().width
-          + (parseInt(teStyles.marginLeft, 10)
-          + parseInt(teStyles.marginLeft, 10));
-        // Set the ContextualHost width
-       
-      } else {
-        this._alignmentWidth = this._alignmentClone.getBoundingClientRect().width
-          + (parseInt(_alignmentStyles.marginLeft, 10)
-          + parseInt(_alignmentStyles.marginRight, 10));
-         this._alignmentClone.setAttribute("style", "");
-      }
-      
-      this._alignmentHeight = this._alignmentClone.getBoundingClientRect().height
-        + (parseInt(_alignmentStyles.marginTop, 10)
-        + parseInt(_alignmentStyles.marginBottom, 10));
-     
-      this._alignmentClone.classList.remove(ALIGNMENT_STATE_POSITIONED);
-      this._alignmentClone.classList.remove(CONTEXT_STATE_CLASS);
-      this._teWidth = this._targetElement.getBoundingClientRect().width;
-      this._teHeight = this._targetElement.getBoundingClientRect().height;
-    }
+
+    // private _savealignmentSize(): void {
```

Comment:
Same feedback as above--is there a reason this is commented out? The code looks like it's been cleaned up.